cypress/e2e/Sitemap/sitemap.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Blog series RSS Feed
+
+    Scenario: Verify the sitemap is loaded
+        Given user requests the sitemap from "/sitemap.xml"
+        Then the response content type is "application/xml; charset=utf-8"
+        And the sitemap has at least 489 urls
+        # make sure we didn't get an infinite sitemap
+        And the sitemap has no more than 900 urls
+        And all sitemap entries start with www.cancer.gov
+
+
+    Scenario Outline: All expected content types appear in the sitemap
+        Given user requests the sitemap from "/sitemap.xml"
+        Then the sitemap includes the path "<path>"
+
+        Examples:
+            ## It is expected that all site map entries point to www.cancer.gov as this is the URL we want indexed if a search engine
+            ## indexes a lower-tier server.
+            ## Commented out entries are for types which are excluded from sitemaps
+            | path                                                                              | type                           |
+            | https://www.cancer.gov/                                                           | Home and Landing page          |
+            | https://www.cancer.gov/espanol                                                    |                                |
+            #| https://www.cancer.gov/search/results                                               | Application page              |
```

Comment:
are commented out entries intended to be enabled in the future? If not, then its better to remove them all together